src/main/java/undo/UndoController.java

```diff
@@ -73,6 +77,11 @@ private boolean handleActionResult(TurboIssue issue, Action<TurboIssue> action,
         return success;
     }
 
+    private boolean handleActionException(TurboIssue issue, Action<TurboIssue> action, Throwable exception) {
+        showErrorDialog(issue, action, Utility.removeFirstWord(exception.getMessage()));
```

Comment:
Maybe we can add a comment here saying what exception messages look like, and what the intended output is. Also, I'm not sure if messages are consistent across exception types. Might need a regex.